prisma/mongodb/schema.prisma

```diff
@@ -17,14 +17,16 @@ model Question {
     updatedAt DateTime @updatedAt
     title     String
     body      String
-    answers   Answer[]
+    difficulty Int
+    category  String
+    // answers   Answer[]
 }
 
-model Answer {
```

Comment:
https://blog.panoply.io/mongodb-best-practices
For nosql databases there isn't a need for us to denormalize data especially if we are gonna query it together generally. I'm not too familiar with mongodb but when i worked with dynamodb one table was recommended. I read the first few paragraphs of the above best practices and got the same vibe. We could consider putting both question and answer into the same table

Reply:
Agree